node/tests/data/packages/git/package-lock.v1.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
very nice to see the tests for this!

Reply:
Thanks, getting all the tests cases across all the various npm versions and lockfiles was quite an ordeal!